setup.py

```diff
@@ -127,19 +132,17 @@ def get_extras_require() -> Dict[str, List[str]]:
             "keras",
             "tensorflow",
             "tensorflow-datasets",
+            "pytorch-ignite",
+            "pytorch-lightning>=0.8.1",
+            "skorch",
+            "catalyst",
         ]
         + (
-            (
-                ["torch==1.6.0", "torchvision==0.7.0"]
-                if sys.platform == "darwin"
-                else ["torch==1.6.0+cpu", "torchvision==0.7.0+cpu"]
-            )
-            + ["pytorch-ignite", "pytorch-lightning>=0.8.1", "skorch"]
-            if (3, 5) < sys.version_info[:2]
-            else []
+            ["torch==1.6.0", "torchvision==0.7.0"]
+            if sys.platform == "darwin"
+            else ["torch==1.6.0+cpu", "torchvision==0.7.0+cpu"]
```

Comment:
A function below can DRY the code a bit (should be out of scope of this PR though).

```python
def get_torch():
    return (
        ["torch==1.6.0", "torchvision==0.7.0"]
        if sys.platform == "darwin"
        else ["torch==1.6.0+cpu", "torchvision==0.7.0+cpu"]
    )

```

Reply:
Sure, that might reduce maintenance cost.